doc/databases.md

```diff
@@ -21,6 +21,11 @@ This will create the tables and relations.
 
 It will also create initial data from the seeds `directory`. This is split up in default and enviroment data. A file `local.js` here will be run, if available.
 
+Since databases are connected (e.g. the API needs to know clients info on the auth server) these scripts are also acvailable from the headless root directory:
```

Comment:
```suggestion
Since databases are connected (e.g. the API needs to know clients info on the auth server) these scripts are also available from the headless root directory:
```